test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -233,13 +243,10 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 		{
 			if (info.name.find("PUSH") != std::string::npos)
 				code += toCompactHex(opcode);
-			code += fillArguments((dev::eth::Instruction) opcode, _options);
-		}
+			code += fillArguments(inst, _options);
 
-		if (info.name.find("PUSH") == std::string::npos)
-		{
-			std::string byte = toCompactHex(opcode);
-			code += (byte == "") ? "00" : byte;
+			if (info.name.find("PUSH") == std::string::npos)
```

Comment:
Well, here's the suggestion to make this clearer
```
if (info.name.find("PUSH") != std::string::npos)
{
  code += toCompactHex(opcode, HexPrefix::DontAdd, 1);
  code += fillArguments(inst, _options);
}
else
{
  code += fillArguments(inst, _options);
  code += toCompactHex(opcode, HexPrefix::DontAdd, 1);
}
```
(or maybe even move the code for generating args to PUSH from `fillArguments` into another function, because it's weird that the same function should be called in different order for different use cases)